definitions/tools/arriba.wdl

```diff
@@ -0,0 +1,44 @@
+version 1.0
+
+task arriba {
+  input {
+    File aligned_bam
+    File reference_annotation
+    File reference
+    File reference_fai
+    File reference_dict
+  }
+
+  Float bam_size_gb = size([aligned_bam, reference_annotation], "GB")
+  Float reference_size_gb = size([reference, reference_fai, reference_dict], "GB")
+  Int space_needed_gb = 10 + round(3*bam_size_gb + reference_size_gb)
+
+  runtime {
+    preemptible: 1
+    maxRetries: 2
+    memory: "64GB"
+    docker: "uhrigs/arriba:2.4.0"
+    disks: "local-disk ~{space_needed_gb} HDD"
+  }
+
+  # explicit typing required, don't inline
+  command <<<
+    /arriba_v2.4.0/arriba \
+    -b /arriba_v2.4.0/database/blacklist_hg38_GRCh38_v2.3.0.tsv.gz \
```

Comment:
I think all these file names need to be updated to reflect `2.4.0` instead of `2.3.0`.  It seems the name of the dir AND the file names change when the db is updated